app/src/main/java/io/github/mattpvaughn/chronicle/features/settings/SettingsViewModel.kt

```diff
@@ -553,6 +553,24 @@ class SettingsViewModel(
                         _webLink.postEvent("https://www.reddit.com/r/ChronicleApp")
                     }
                 }),
+            PreferenceModel(
+                type = PreferenceType.CLICKABLE,
+                title = FormattableString.from(R.string.settings_github_title),
+                explanation = FormattableString.from(R.string.settings_github_explanation),
+                click = object : PreferenceClick {
+                    override fun onClick() {
+                        _webLink.postEvent("https://github.com/mattttvaughn/chronicle")
+                    }
+                }),
+            PreferenceModel(
+                type = PreferenceType.CLICKABLE,
+                title = FormattableString.from(R.string.settings_version_title),
+                explanation = FormattableString.from(BuildConfig.VERSION_NAME),
+                click = object : PreferenceClick {
```

Comment:
don't think we need this click here. but nothing to sweat, isn't hurting anything.

I'm happy to merge this in as it is as long as you weren't planning on using it for anything

Reply:
I looked at the available options under PreferenceType and I didn't see a better option. I'm happy to change it, I wasnt planning on using it.